torch/_tensor_docs.py

```diff
@@ -5378,6 +5378,47 @@ def callable(a, b) -> number
     tensor(indices=tensor([[1]]),
            values=tensor([[ 9,  0, 10]]),
            size=(3, 3), nnz=1, layout=torch.sparse_coo)
+
+.. method:: to_sparse(*, layout=None, blocksize=None) -> Tensor
+   :noindex:
+
+Returns a sparse tensor with the specified layout and blocksize.
+
+.. note:: If the :attr:`self` layout and blocksize parameters match
+          with the specified layout and blocksize, return
+          :attr:`self`. Otherwise, return a sparse tensor copy of
+          :attr:`self`.
+
+Args:
+
+    layout (:class:`torch.layout`, optional): The desired sparse
+      layout. One of ``torch.sparse_coo``, ``torch.sparse_csr``,
+      ``torch.sparse_csc``, ``torch.sparse_bsr``, or
+      ``torch.sparse_bsc``. Default: if ``None``,
+      ``torch.sparse_coo``.
+
+    blocksize (list, tuple, :class:`torch.Size`, optional): Block size
+      of the resulting BSR or BSC tensor. For other layouts,
+      specifying the block size that is not ``None`` will result a
+      RuntimeError exception.  A block size must be a 2-sequence and
```

Comment:
Can't we just say "tuple of length two" and "evenly divide the sparse dimensions"?

Reply:
How about this:
```
A block size must be a tuple of length two such that its items are evenly divisible with the two sparse dimensions.
```
?

Reply:
Fixed.